packages/app/src/components/Bookmarks/BookmarkItem.tsx

```diff
@@ -106,34 +151,37 @@ const BookmarkItem = (props: Props): JSX.Element => {
             onPressEnter={pressEnterForRenameHandler}
             inputValidator={inputValidator}
           />
+
         ) : (
-          <a href={`/${bookmarkedPage._id}`} className="grw-foldertree-title-anchor flex-grow-1 pr-3">
-            <p className={`text-truncate m-auto ${bookmarkedPage.isEmpty && 'grw-sidebar-text-muted'}`}>{pageTitle}</p>
-          </a>
+          <>
+            <div className='d-flex flex-row'>
+              <PageListItemS page={bookmarkedPage} />
+              <PageItemControl
+                pageId={bookmarkedPage._id}
+                isEnableActions
+                forceHideMenuItems={[MenuItemType.DUPLICATE]}
+                onClickBookmarkMenuItem={bookmarkMenuItemClickHandler}
+                onClickRenameMenuItem={renameMenuItemClickHandler}
+                onClickDeleteMenuItem={deleteMenuItemClickHandler}
+              >
+                <DropdownToggle color="transparent" className="border-0 rounded btn-page-item-control p-0 grw-visible-on-hover mr-1">
+                  <i className="icon-options fa fa-rotate-90 p-1"></i>
+                </DropdownToggle>
+              </PageItemControl>
+            </div>
+          </>
         )}
-        <PageItemControl
-          pageId={bookmarkedPage._id}
-          isEnableActions
-          forceHideMenuItems={[MenuItemType.DUPLICATE]}
-          onClickBookmarkMenuItem={bookmarkMenuItemClickHandler}
-          onClickRenameMenuItem={renameMenuItemClickHandler}
-          onClickDeleteMenuItem={deleteMenuItemClickHandler}
-        >
-          <DropdownToggle color="transparent" className="border-0 rounded btn-page-item-control p-0 grw-visible-on-hover mr-1">
-            <i className="icon-options fa fa-rotate-90 p-1"></i>
-          </DropdownToggle>
-        </PageItemControl>
-        <UncontrolledTooltip
-          modifiers={{ preventOverflow: { boundariesElement: 'window' } }}
-          autohide={false}
-          placement="right"
-          target={bookmarkItemId}
-          fade={false}
-        >
-          { formerPagePath !== null ? `${formerPagePath}/` : '/' }
-        </UncontrolledTooltip>
+
       </li>
-    </div>
+    );
+  };
+  return (
+    <>
+      {isSidebarItem
+        ? renderBookmarkListForSidebar()
+        : renderBookmarkListForUserPage()
+      }
```

Comment:
Could you use the same component for each Bookmark contents? (`Sidebar` and `/me`).
If you think it's impossible, please tell me why.

especially, bookmarks tree is not difference between sidebar and /me